tests/unit/utils/concurrent-axe-test.js

```diff
@@ -1,72 +1,72 @@
 /* global sinon, axe */
-import { module, test } from 'ember-qunit';
-import wait from 'ember-test-helpers/wait';
+import { module, test } from 'qunit';
 import { ConcurrentAxe } from 'ember-a11y-testing/utils/concurrent-axe';
+import wait from 'ember-test-helpers/wait';
```

Comment:
We should be using `import { settled } from '@ember/test-helpers` instead of `wait`